### PR TITLE
PP-6152 Add metadata to payment link page

### PR DESCRIPTION
**Pull request by sfount**
* resource controllers for GETting add metadata to payment link page
* initial template for column header and cell value form
* restrict access to pages and POST routes by `experimental_features_enabled` flag

### CODE REVIEW

File: app/controllers/payment-links/metadata/resource-controller.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const { response } = require('../../../utils/response')
```
Comment (kbottla) [inline app/controllers/payment-links/metadata/resource-controller.js]: Would be useful to have tests for this controller ?

File: app/paths.js [hunk 2]
```diff
@@ -150,7 +150,10 @@ module.exports = {
     edit: '/create-payment-link/manage/edit/:productExternalId',
     editInformation: '/create-payment-link/manage/edit/information/:productExternalId',
     editReference: '/create-payment-link/manage/edit/reference/:productExternalId',
-    editAmount: '/create-payment-link/manage/edit/amount/:productExternalId'
+    editAmount: '/create-payment-link/manage/edit/amount/:productExternalId',
+    metadata: {
+      add: '/create-payment-link/manage/edit/:productExternalId/metadata'
```
Comment (kbottla) [inline app/paths.js]: I guest path is `create-payment-link` due to existing convention ?

File: app/controllers/payment-links/metadata/resource-controller.js (hunk 1 above)
Comment (sfount) [reply app/controllers/payment-links/metadata/resource-controller.js]: I think so! it's currently used by every controller though and don't think it fits into the scope of this PR. We could do this is a BAU improvement (although I would like to stop passing `req`, `res` etc. down into this library and refactor it to just provide a context).

File: app/paths.js (hunk 2 above)
Comment (sfount) [reply app/paths.js]: yeah, maybe we should rethink that once we get all the 'management' pages in

**Pull request merged**